CHANGES.md

```diff
@@ -131,6 +132,9 @@
   --temp_location, or pass method="STREAMING_INSERTS" to WriteToBigQuery ([BEAM-6928](https://issues.apache.org/jira/browse/BEAM-6928)).
 * Python SDK now understands `typing.FrozenSet` type hints, which are not interchangeable with `typing.Set`. You may need to update your pipelines if type checking fails. ([BEAM-10197](https://issues.apache.org/jira/browse/BEAM-10197))
 
+## Known issues
+
+* When a timer fires but is reset prior to being executed, a watermark hold may be leaked, causing a stuck pipeline [BEAM-10991](https://issues.apache.org/jira/browse/BEAM-10991).
 
```

Comment:
Are we missing one line here?

In https://github.com/apache/beam/pull/12997 we have
`Dataflow streaming timers are not strictly time ordered when set earlier mid-bundle ([BEAM-8543](https://issues.apache.org/jira/browse/BEAM-8543)).`

Reply:
Thorough review! Actually I did a pure revert in #12980 so I reverted that line from `CHANGES.md`. Then in #12997 I added it back. If you "view file" at this commit you will see it is there, because on the release branch it was not accidentally reverted.

Reply:
Ah I see. I didn't see this line was reverted. Thanks!